extension/services/spotify/player.js

```diff
@@ -11,8 +11,20 @@ this.player = (function() {
     }
 
     async action_search({ query, thenPlay }) {
+      // try to find the error page; if found, throw a DRM error; otherwise play
+      try {
+        this.querySelector("div.ErrorPage");
```

Comment:
This doesn't test anything! This will return the element if it is found, or `null` if it is not found. So you need to have a test and throw the exception based on that test.

Reply:
Perhaps I'm misunderstanding, but I thought `this.querySelector` throws an error if it nulls out? Should I be using `document.querySelector` instead?

Reply:
Oh... you are right, I missed that. Either using `this.querySelector` and catching it, or checking for null with `document.querySelector` would be fine.